### DISCUSSION TITLE
Welcome to VBA-SQL-Library Discussions!

### DISCUSSION THREAD

**Discussion by Beakerboy** [Beakerboy/VBA-SQL-Library#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by winner106** [comment]
if Excel Table can be a database, that would be cool~

**Comment by Beakerboy** [comment]
Excel has certain database functions, like DSum. However, using Excel as a database is a horrible idea. That’s what Access is for, and Mysql, MariaDB, or PostgreSQL are even better.
